lib/Makefile.uk

```diff
@@ -70,3 +70,4 @@ $(eval $(call import_lib,$(CONFIG_UK_BASE)/lib/ukrust))
 $(eval $(call import_lib,$(CONFIG_UK_BASE)/lib/ukreloc))
 $(eval $(call import_lib,$(CONFIG_UK_BASE)/lib/ukofw))
 $(eval $(call import_lib,$(CONFIG_UK_BASE)/lib/ukallocstack))
+$(eval $(call import_lib,$(CONFIG_UK_BASE)/lib/ukbinfmt))
```

Comment:
> lib: Introduce libukbinfmt

s/sha-bang/shebang in the commit msg

Reply:
"sha-bang" is valid, see https://en.wikipedia.org/wiki/Shebang_(Unix)

Reply:
Ok fine... but I think you can agree with me that we all would much rather have a "she" than a "sha" :innocent: :stuck_out_tongue: